src/generateDocumentation.js

```diff
@@ -292,12 +292,33 @@ function generateModels(templateModels) {
 
   templateModels.forEach((templateModel) => {
     if (!templateModel.name || !templateModel.schema) return;
-    models[templateModel.name] = templateModel.schema;
+    models[templateModel.name] = generateModelSchema(templateModel.schema);
   });
 
   return models;
 }
 
+function generateModelSchema(templateSchema) {
+  if (!templateSchema) return templateSchema;
+
+  let schema = {};
+  Object.keys(templateSchema).forEach((key) => {
+    let field = templateSchema[key];
+    if (key === "$ref") {
+      const match = /{{\s*model\s*:\s*([\-\w]+)\s*}}/.exec(field);
+      if (match) {
+        field = `#/components/schemas/${match[1]}`;
+      }
+    } else if (field.constructor === Object && field) {
```

Comment:
A bit of a nit pick... Can we change the order. Makes more sense.

```
} else if (field && field.constructor === Object) {
```

Reply:
Sure. Changed!